src/mgr/PythonCompat.h

```diff
@@ -25,4 +28,7 @@ inline PyObject* PyInt_FromLong(long ival) {
 inline int PyString_Check(PyObject *o) {
   return PyUnicode_Check(o);
 }
+inline PyObject PyFloat_FromString(PyObject *s, void *arg) {
```

Comment:
This needs to return `PyObject*`

Reply:
Dammit, we also need this:

````
inline PyObject* PyInt_FromString(const char *str, char **pend, int base) {
  return PyLong_FromString(str, pend, base);
}
````

Reply:
...and this:

````
#define PyString_Type PyUnicode_Type
````

With these three changes, it'll build just fine against Python 3.